apps/examples/src/examples/custom-config/CardShape/card-shape-props.ts

```diff
@@ -1,8 +1,8 @@
-import { DefaultColorStyle, ShapeProps, T } from 'tldraw'
+import { DefaultColorStyle, RecordProps, T } from 'tldraw'
```

Comment:
`ShapeProps` seems like a good and easy target for graceful deprecation: lots of people probably using it and zero complexity to avoid breaking. we can keep examples using RecordProps, but let's just export an alias of it somewhere with a `@deprecated` tag?

Reply:
ah ignore me, there are a few other types in this kinda Shape* namespace, might as well just rip off the band aid.